### PR TITLE
setup rke2 and rancher in vm

### PR DESCRIPTION
**Pull request by ivan-claire**

### CODE REVIEW

**Comment by kravciak** [comment]
This won't work, scripts are not self-sufficient but part of validator testsuite. You will miss libraries, env setup, variables...

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,40 +17,60 @@ jobs:
...
-          terraform validate 
-          terraform apply --auto-approve
-      # TODO: Setup RKE2 Cluster
-      # TODO: Run Helm chart linter
-      # TODO: Run Basic tests for specific chart-name
+          cd openstack_env/scripts
+          cp  ~/developer-s390-openrc.sh env.conf
+          chmod 400 data/id_shared
+          sed -i 's/\-it//g' varke 
+          ./varke run -n 1:0 -t longhorn -e SUFFIX=s390x-charts-lh -k
```
Comment (kravciak) [inline .github/workflows/ci.yml]: If you remove `-k` cluster (terraform) will destroy automatically after test runs.
Then you can remove destroy-vm step.

Comment (ivan-claire) [reply .github/workflows/ci.yml]: yeah, I put the -k intentionally so if someone wants to rerun the pipeline and check or cat the logs on this step before destroying the vm, they can do so.

**Review by thehejik — APPROVED** [review]
LGTM

**Pull request merged**